src/classes/Defaults/Carrier/Webhook.php

```diff
@@ -41,6 +41,7 @@ public function form_fields() {
 		] ) );
 
 		$this->add_form_field( new Field\RepeaterField( [
+			'carrier'          => 'webhook',
```

Comment:
This is a compatibility break. Can't you do this automatically while adding the field to the carrier?